contracts/governance/Staking/Staking.sol

```diff
@@ -390,6 +549,25 @@ contract Staking is
         }
     }
 
+    /**
+     * @dev withdraws tokens for lock date 2 weeks later than given lock date
+     * Dedicated only for direct withdrawal from staking contract.
+     */
+    function _withdrawNextDirect(
```

Comment:
delete

Reply:
deleted in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/19ef0dedeffdf80545269ffafe2f1af684a25a06)